packages/components/src/FormField/FormFieldTypes.ts

```diff
@@ -91,7 +84,12 @@ export interface CommonFormFieldProps {
    * Simplified onChange handler that only provides the new value.
    * @param newValue
```

Comment:
need to add another param
```suggestion
   * @param newValue
   * @param event
```

Reply:
Ahh right

Reply:
https://github.com/GetJobber/atlantis/pull/783/commits/76b87d32cb43beceec905435da979776985e3003